wiki/Hand-tracking-controller-bindings.md

```diff
@@ -1,38 +1,17 @@
-Current bindings for ALVR 14.0.0
+Current bindings for ALVR 20
```

Comment:
You can add a line saying how to disable gestures.

Reply:
Added couple of lines on how to enable/disable it